kotlinx-coroutines-core/api/kotlinx-coroutines-core.api

```diff
@@ -1020,6 +1020,13 @@ public final class kotlinx/coroutines/flow/internal/SendingCollector : kotlinx/c
 	public fun emit (Ljava/lang/Object;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 }
 
+public final class kotlinx/coroutines/flow/operators/ChunkKt {
+	public static final fun chunked (Lkotlinx/coroutines/flow/Flow;I)Lkotlinx/coroutines/flow/Flow;
+	public static final fun chunked (Lkotlinx/coroutines/flow/Flow;ILkotlin/jvm/functions/Function2;)Lkotlinx/coroutines/flow/Flow;
+	public static final fun windowed (Lkotlinx/coroutines/flow/Flow;IIZ)Lkotlinx/coroutines/flow/Flow;
+	public static final fun windowed (Lkotlinx/coroutines/flow/Flow;IIZLkotlin/jvm/functions/Function2;)Lkotlinx/coroutines/flow/Flow;
+}
+
```

Comment:
This needs to be updated with `apiDump`.  All the operation should now be in `FlowKt`, not in `Chunk.kt`.

Reply:
Oh, sorry! Obsolete RingBuffer removed. ApiDump generated from scratch, should be all right now.